marketplace-ui/src/app/modules/product/product-detail/product-detail.component.ts

```diff
@@ -380,4 +387,19 @@ export class ProductDetailComponent {
     const value = key.value as tabName;
     return this.productModuleContent()[value];
   }
-}
+
+  private setDefaultVendorImage(productDetail: ProductDetail): ProductDetail {
+    const { vendorImage, vendorImageDarkMode } = productDetail;
+
+    if (!productDetail.vendorImage && !productDetail.vendorImageDarkMode) {
```

Comment:
I believe `and` is correct, in case both didn't have value then it should be assigned default value for both. And if only one has value then both image should the same. Last case is both has value, it always get the first value (get `a` in expression a || b)